Examples/create-windows-cloud-image.ps1

```diff
@@ -12,39 +12,70 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-$ErrorActionPreference = "Stop"
-
 git submodule update --init
-Import-Module ..\WinImageBuilder.psm1
+Import-Module ..\WinImageBuilder.psm1 -Verbose
+Import-Module ..\Config.psm1 -Verbose
+# The path were you want to create the config fille
+$ConfigFilePath ="C\image\config.ini"
```

Comment:
by doing this, people can reason better about the changes needed to accommodate their old scripts to the new format